great_tables/_formats.py

```diff
@@ -3713,7 +3707,7 @@ def fmt_nanoplot(
         # Why are we grabbing the first element of a tuple? (Note this also happens again below.)
         all_single_y_vals = list(
             data_tbl[columns].apply(
```

Comment:
We should maybe take this `.apply` out at some point, but because pandas and polars implement it in a similar what it works okay 😁